unified-codes/libs/ui/src/lib/components/entity/molecules/EntityDetailList.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { Property, IEntity } from '@unified-codes/data';
+import { Typography } from '../../data';
+import { Grid } from '../../layout';
+import { Link } from '../../navigation';
+import { EntityDetailCategoryItem } from './EntityDetailCategoryItem';
+import { TypographyVariant } from '@material-ui/core';
+
+export interface IExternalLink {
+  type: string;
+  url: string;
+}
+
+export interface EntityDetailListProps {
+  headerVariant?: TypographyVariant;
+  productSubCategories?: IEntity[];
+  entityProperties?: Property[];
+  externalLinks?: IExternalLink[];
+}
+
+export type EntityDetailList = React.FunctionComponent<EntityDetailListProps>;
+
+export const EntityDetailList: EntityDetailList = ({
```

Comment:
I think this component is probably useful, but again must be annoying and say it contains logic 😢. Ideally library components should just take props and render them, not derive data and stuff.